apps/srt-live-transmit.cpp

```diff
@@ -136,6 +136,7 @@ struct LiveTransmitConfig
     bool log_internal;
     string logfile;
     int bw_report = 0;
+    bool srctime = true;
```

Comment:
Given we recommend using the source time with monotonic or C++11 clock, would be better to disable it by default.

Reply:
We provide this monotonic clock in the `srt_time_now()` function. Still, I've been rather predicting this function to be provided as "boolean per presence", so if this has to be on by default, it should definitely use a different name.